xbmc/windowing/osx/WinSystemOSX.mm

```diff
@@ -1028,16 +1024,19 @@ static void DisplayReconfigured(CGDirectDisplayID display,
     bool stretched;
     bool interlaced;
     bool safeForHardware;
-    size_t w;
-    size_t h;
+    size_t resWidth;
+    size_t resHeight;
+    size_t pixelWidth;
+    size_t pixelHeight;
     size_t bitsperpixel;
     double refreshrate;
     RESOLUTION_INFO res;
 
     CFArrayRef displayModes = GetAllDisplayModes(GetDisplayID(disp));
-    NSString* const dispName = screenNameForDisplay(disp);
+    const char* dispName = [screenNameForDisplay(disp) UTF8String];
```

Comment:
Better keep the objc string because utf8string returns internal pointer, rather keep the object alive explicitly